src/backend/main/py.main/caliopen_main/user/store/user.py

```diff
@@ -74,11 +74,12 @@ class RemoteIdentity(BaseModel):
     """User remote identities model."""
 
     user_id = columns.UUID(primary_key=True)
-    identity_id = columns.Text(primary_key=True)
+    identifier = columns.Text(primary_key=True)
+    display_name = columns.Text()
     type = columns.Text()
     status = columns.Text()
-    credentials = columns.List(columns.Text)
     last_check = columns.DateTime()
+    infos = columns.Map(columns.Text, columns.Text)
```

Comment:
OK in fact I'll integrate vault service (by hashicorp) to store these sensitives informations.

In RemoteIdentity this field is no more needed, the related informations/credentials,server,port,etc will be stored in vault under a remote_id key

WDYT ? we've got a way to secure correctly such sensitive informations